lib/stripe.ex

```diff
@@ -17,6 +17,19 @@ defmodule Stripe do
 
       config :stripity_stripe, api_key: System.get_env("STRIPE_API_KEY")
 
+  ### Shared Options
+
+  Almost all of the requests that can be sent accept the following options:
+
+    * `:api_key` - The Stripe API key to use for the request. See
+      [https://stripe.com/docs/api/authentication](https://stripe.com/docs/api/authentication)
+    * `:connect_account` - A Stripe Connect account ID which should originate
+      the specified action using the "Stripe-Account" header. The preferred
```

Comment:
`should originate the specified action` - any different wording that could be applied here?

Reply:
That's great feedback. What would you think of this?
```
The ID of a Stripe Connect account for which the request should be made, passed through as the "Stripe-Account" header. The preferred authentication method for Stripe Connect.
```

Reply:
Just updated this. What do you think of the new one?
```
* `:connect_account` - The ID of a Stripe Connect account for which the
  request should be made, passed through as the "Stripe-Account" header. The
  preferred authentication method for Stripe Connect. See
  [https://stripe.com/docs/connect/authentication#stripe-account-header](https://stripe.com/docs/connect/authentication#stripe-account-header)
```